packages/css-data/src/tailwind-parser/shorthand.ts

```diff
@@ -0,0 +1,75 @@
+export const expandShorthand = (property: string) => {};
+
+/**
+ * Expands shorthand CSS properties into their long-form equivalents.
+ * For example, 'mx' is expanded to 'mr' (margin-right) and 'ml' (margin-left).
+ **/
+export const expandTailwindShorthand = (classname: string) => {
+  let expanded = conflictingClassGroups[classname];
+
+  if (expanded) {
+    return expanded.join(" ");
+  }
+
+  const parts = classname.split("-");
+  const modifier = parts.pop();
+  const base = parts.join("-");
+
+  expanded = conflictingClassGroups[base];
+
+  if (expanded) {
+    return expanded.map((cls) => `${cls}-${modifier}`).join(" ");
+  }
+
+  return classname;
+};
+
+/**
+ * A little bit changed tailwind-merge
+ * conflictingClassGroups https://github.com/dcastil/tailwind-merge/blob/fb1fc1f9d106f2184e24949e6e6d1a67904706fa/src/lib/default-config.ts#L1716
+ *
+ * Attributions
+ * MIT License
+ * Copyright (c) 2021 Dany Castillo
+ **/
+const conflictingClassGroups: Record<string, string[]> = {
```

Comment:
is there a chance tailwind will evolve and we would have to keep up with this function? wish we would take what they use

Reply:
Its our internal details that we don't support shorthands